linera-core/src/chain_worker/state.rs

```diff
@@ -440,7 +440,10 @@ where
         let targets = self.chain.outboxes.indices().await?;
         let outboxes = self.chain.outboxes.try_load_entries(&targets).await?;
         for (target, outbox) in targets.into_iter().zip(outboxes) {
-            let heights = outbox.queue.elements().await?;
+            let heights = match outbox {
+                Some(outbox) => outbox.queue.elements().await?,
+                None => Vec::new(),
```

Comment:
Just for my understanding: This should be unreachable, right? Also the `if let Some(…)` below should always be `Some`?

Would it make sense to have a `try_load_all_entries` or something? We seem to be using that in a few places.